config-templates/tables.postgres.sql

```diff
@@ -0,0 +1,38 @@
+CREATE TABLE "statistics_sums" (
```

Comment:
I would personally type the types in UPPER_CASE as well, but leave that to your choice.